webui-src/app/scss/pages/_files.scss

```diff
@@ -178,3 +182,115 @@ table.friendsfiles td:nth-child(2) {
     margin-left: 0.5rem;
   }
 }
+
+.shareManagerPopupOverlay {
+  @include popupOverlay;
+  .shareManagerPopup {
+    position: absolute;
+    inset: 0;
+    margin: auto;
+    width: 80%;
+    height: 90%;
+    & > .widget {
+      padding: 1.5rem;
+    }
+    & .close-btn {
+      position: absolute;
+      top: 1.5rem;
+      right: 1.5rem;
+    }
+  }
+}
+
+.share-manager {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  &__table {
+    margin: 1rem 0 auto;
+    thead {
+      font-weight: bold;
+      td:nth-child(3),
+      td:nth-child(4) {
+        text-align: left;
```

Comment:
I feel like we should align:left all headings, the first & second headings look a bit out of place, what do you think?

Reply:
Yeah, but I also did the same in start, but the table cells are input elements and have some padding, so It doesn't look good.
![1694231972_Sep09_09:29:32](https://github.com/RetroShare/RSNewWebUI/assets/93514460/5be71899-5577-4490-93ba-b30c36385b2c)

I can maybe add some padding to the headings as well and then do it. Is that good?
![1694232295_Sep09_09:34:55](https://github.com/RetroShare/RSNewWebUI/assets/93514460/cfcd5c10-88fa-4cd9-9dad-aaf762f7225b)

Reply:
yeah that sounds better